### PR TITLE
Release v1.2.3

### PR DESCRIPTION
**Pull request by lifubang**
## [1.2.3] - 2024-12-12

> Winter is not a season, it's a celebration.

### Fixed
 * Fixed a regression in use of securejoin.MkdirAll, where multiple
   runc processes racing to create the same mountpoint in a shared rootfs
   would result in spurious EEXIST errors. In particular, this regression
   caused issues with BuildKit. (#4543, #4550)
 * Fixed a regression in eBPF support for pre-5.6 kernels after upgrading
   Cilium's eBPF library version to 0.16 in runc. (#3008, #4551)

### CODE REVIEW

**Review by cyphar — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+> Winter is not a season, it's a celebration.
+
+### Fixed
+ * Fixed a regression in use of securejoin.MkdirAll, where multiple
+   runc processes racing to create the same mountpoint in a shared rootfs
+   would result in spurious EEXIST errors. In particular, this regression
+   caused issues with BuildKit. (#4543, #4550)
+ * Fixed a regression in eBPF support for pre-5.6 kernels after upgrading
+   Cilium's eBPF library version to 0.16 in runc. (#3008, #4551)
```
Comment (thaJeztah) [inline CHANGELOG.md:19]: Definitely not for this PR, but these ticket / PR numbers only link when viewed on github and inside this repository. If we want them to be functional as part of the downloaded source / module, we should perhaps bit the bullet once, and update them to be proper links.

**Review by thaJeztah — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (lifubang) [reply CHANGELOG.md:19]: I’ll update later, let’s do it from this PR.

**Review by kolyshkin — APPROVED** [review]
LGTM

**Pull request merged**